.travis.yml

```diff
@@ -35,6 +40,7 @@ jobs:
         - curl "${GRAVIS}.install-jdk-travis.sh"
           --output ~/.install-jdk-travis.sh
         - source ~/.install-jdk-travis.sh
+        - npm install --no-save
```

Comment:
Why are we doing this?

Reply:
I would rather avoid committing package.json just for csslint.

Reply:
But, to get this css lint work we need this package.json and run this commans. I found this from an existing coala repository https://github.com/coala/coala-html/blob/master/package.json

Reply:
Yes, but we don't need a package.json for this repo. So, better just add `npm install csslint` in travis.yml to get the csslint executable.

Reply:
Also by that I don't think we'd need the manual bundler install. Will need to check this though, so try removing that bundler change and push with just csslint install. If that still breaks, we can add the custom bundler install command again.

Reply:
Removed last commit and package.json too. Build failed due to `bundler` :disappointed: